demo/index.html

```diff
@@ -17,13 +17,20 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
     <link rel="import" href="../../paper-styles/typography.html">
-    <link rel="import" href="x-translate.html">
-    <link rel="import" href="x-local-translate.html">
 
     <!-- Intl polyfill -->
     <script src="../../intl/dist/Intl.min.js"></script>
     <script src="../../intl/locale-data/jsonp/en.js"></script>
+
+    <!-- Must import polymer before running next script. -->
+    <link rel="import" href="../../polymer/polymer.html">
+    <!-- Set this before importing components that use app-localize-behavior. -->
+    <script>Polymer.defaultLanguage = 'fr'</script>
```

Comment:
As mentioned, I'm not super keen on this, as this is not how we set default values on elements